components/bsx/Offer/OfferList.vue

```diff
@@ -76,15 +79,20 @@ export default class OfferList extends mixins(
     this.total = response.stats.total
   }
 
-  protected async fetchOffers() {
+  protected fetchOffers() {
     try {
-      const { data } = await this.$apollo.query<OfferResponse>({
+      this.$apollo.addSmartQuery<OfferResponse>('offers', {
```

Comment:
Why replace `query` with `addSmartQuery` 👀

Reply:
Thus the query can be queried again when variables are update.

```
variables: () => ({
          id: createTokenId(this.collectionId, this.nftId),
          account: this.currentOwnerId,
        }),
```
It is reactive variable using function as return above. However, it can't be used in normal query